Core.ApplicationServices/Model/KitosEvents/SystemDataProcessorChangeEventBodyModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+namespace Core.ApplicationServices.Model.KitosEvents;
+
+public class SystemDataProcessorChangeEventBodyModel : IEventBody
+{
+    public Guid SystemUuid { get; set; }
+    public Guid? DataProcessorUuid { get; set; }
+    public string DataProcessorName { get; set; }
+
+    public string Type()
+    {
+        return "DataProcessorChange";
```

Comment:
I think a SystemEventTypes enum should be introduced. To get the name here we can then call nameof(SystemEventTypes.DataProcessorChange). 

Or

The event names should be moved to a consts file. So then we can get the name by calling Consts.Events.SystemUsage.DataProcessorChange (by nesting the classes)